### PR TITLE
Update CODEOWNER for Nick F

### PR DESCRIPTION
**Pull request by bitzoic**
## Type of change

- Improvement (refactoring, restructuring repository, cleaning tech debt, ...)

## Changes

The following changes have been made:

- Nick is now codeowner

## Notes

- Nick is the interim lead

### CODE REVIEW

File: .github/CODEOWNERS
Comment (Braqzen) [inline .github/CODEOWNERS:1]:
```suggestion
* @nfurfaro @braqzen
```

The codeowners is used for notifications for reviews and I want to get notified. I'm the primary reviewer anyway.
Moreover, it goes beyond what the interim role is meant to do anyway so I suggest speaking to Nick about this first before adding him.

**Review by Braqzen — CHANGES_REQUESTED** [review]

**Review by supiket — APPROVED** [review]
LGTM

**Review by rostyslavtyshko — APPROVED** [review]

File: .github/CODEOWNERS
Comment (nfurfaro) [reply .github/CODEOWNERS:1]: I'm going to check in with management about any policy/directives regarding codeowners.
From my understanding, either individuals or a team may be assigned as codeowners, but that person/team must have write permissions for the repo.

Comment (nfurfaro) [inline .github/CODEOWNERS:1]:
```suggestion
* @FuelLabs/application-dev 
```

Comment (nfurfaro) [reply .github/CODEOWNERS:1]: I think making the team the owners of the code could work.

Comment (Braqzen) [reply .github/CODEOWNERS:1]: I'm happy to approve if they accept your commit suggestion.

**Review by Braqzen — APPROVED** [review]

**Pull request merged**
